plugin/eqdetect/source/dextool/plugin/eqdetect/subfolder/dbhandler.d

```diff
@@ -83,8 +83,9 @@ class DbHandler{
         auto stmt = db.prepare(format("SELECT path, lang FROM files WHERE id='%s';", file_id));
         auto res = stmt.execute;
 
+        import std.path;
         auto path = res.front.peek!string(0);
-        path = "../" ~ path; //Ugly solution for now
+        path = buildPath("../" ~ path); //Ugly solution for now
```

Comment:
nice. Further refinement.
tip:
```d
buildPath("..", path);
```

It concatenates paths "correctly". Especielly useful to avoid extra "/" and make it platform independent.